schema/products/uabh.json

```diff
@@ -0,0 +1,313 @@
+            },
+            "providers": {
+              "const": [
+                {
+                  "name": "Copernicus Land Monitoring Service",
+                  "description": "The Copernicus Land Monitoring Service provides geographical information on land cover and its changes, land use, ground motions, vegetation state, water cycle and Earth's surface energy variables to a broad range of users in Europe and across the World in the field of environmental terrestrial applications.",
+                  "roles": ["licensor", "host"],
+                  "url": "https://land.copernicus.eu"
+                }
+              ]
+            },
+            "proj:epsg": { "const": 3035 },
+            "proj:bbox": {
+              "type": "array",
+              "items": {
+                "type": "integer"
+              }
+            },
+            "proj:shape": {
+              "type": "array",
+              "items": {
+                "type": "integer"
+              }
+            }
+          }
+        },
+        "collection": { "const": "urban-atlas-building-height" },
+        "links": {
+          "type": "array",
+          "allOf": [
+            {
+              "contains": {
+                "const": {
+                  "rel": "license",
+                  "href": "https://land.copernicus.eu/en/data-policy"
+                }
+              }
+            },
+            {
+              "contains": {
+                "type": "object",
+                "properties": {
+                  "rel": { "const": "self" }
+                }
+              }
+            },
+            {
+              "contains": {
+                "type": "object",
+                "properties": {
+                  "rel": { "const": "root" }
+                }
+              }
+            },
+            {
+              "contains": {
+                "type": "object",
+                "properties": {
+                  "rel": { "const": "parent" }
+                }
+              }
+            },
+            {
+              "contains": {
+                "type": "object",
+                "properties": {
+                  "rel": { "const": "collection" }
+                }
+              }
+            }
+          ]
+        },
+        "assets": {
+          "type": "object",
+          "minProperties": 4,
+          "maxProperties": 6,
+          "propertyNames": {
+            "anyOf": [
+              { "const": "dataset" },
+              { "const": "quality_check_report" },
+              { "const": "metadata" },
+              { "const": "quality_control_report" },
+              { "type": "string", "pattern": "^pixel_based_info" },
+              { "const": "compressed_dataset" }
+            ]
+          }
+        }
+      }
+    },
+    {
+      "$comment": "This is the schema for Vegetation Phenology and Productivity Collection",
```

Comment:
Give me some context.. VPP is ok here?

Reply:
No, it should be urban atlas building height. Thank you!